.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
```

Comment:
We also have a setup.cfg file than has some flake8 settings. I would recommend keeping all these config settings in that file, so possible conflicts (e.g. file length) can be identified within a single file.

Reply:
That's true, I'll delete .flake8. For the others, I'm afraid that there is not one way to do everything in one file. 

flake8 can live in pyproject.toml and blakc/isort can go to setup.cfg I believe. I did improve the situation a bit. pyproject.tol is the preferred way forward it seems.